sdk/extension-wcf/Microsoft.CoreWCF.Azure.StorageQueues/src/Microsoft.CoreWCF.Azure.StorageQueues.csproj

```diff
@@ -27,6 +27,7 @@
     <PackageReference Include="Azure.Identity" />
     <PackageReference Include="Azure.Storage.Queues" />
     <PackageReference Include="CoreWCF.Queue" />
+    <PackageReference Include="System.Formats.Asn1" />
```

Comment:
Why did this need to be added?

Reply:
Taking ownership of a transitive dependency in the CoreWCF package.   It looks like the newest release moved to the 8.x line, but the `Asn1` package has been flagged for this library on the last 3 reports in different versions, so I'm adding a direct reference so that we can manage the version explicitly.